### DISCUSSION TITLE
Any possibility to run the new Fluent design system of Windows 11 in Windows 10 ?

### DISCUSSION THREAD

**Discussion by oSethoum** [Kinnara/ModernWpf#405] [Q&A]
The new xaml controls library uses the new design system for Windows 11, is it possible to host the controls with WinRT in WPF or UWP app and run it in Window 10 ? 

![image](https://user-images.githubusercontent.com/88779394/138601323-f99ae8e5-5284-48be-af34-61fa0c68756b.png)

**Comment by ghost1372** [comment]
you can use mux in windows 10 and windows 11 also you can host the controls with xaml island
